website/content/api-docs/auth/jwt.mdx

```diff
@@ -112,8 +112,8 @@ entities attempting to login. At least one of the bound values must be set.
 - `name` `(string: <required>)` - Name of the role.
 - `role_type` `(string: <optional>)` - Type of role, either "oidc" (default) or "jwt".
 - `bound_audiences` `(array: <optional>)` - List of `aud` claims to match against.
-  Any match is sufficient. Required for "jwt" roles if the JWT has an `aud`
-  claim. Optional for "oidc" roles.
+  This parameter is **mandatory** for "jwt" roles if the JWT contains an audience (which is more often than not the case),
+  and **must** match at least one of the JWT's associated `aud` claims. Optional for "oidc" roles.
```

Comment:
```suggestion
  The `bound_audiences` parameter is required for "jwt" roles that contain an
  audience (typical case) and **must** match at least one of the associated JWT
  `aud` claims.
```
Style correction: avoid "this" as a pronoun, write with simple, declarative sentences.

I'm assuming the fact that this is the typical case is the important part since it's repeated elsewhere so I left it in, but typically I'd recommend removing it for simplicity.